src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -82,13 +82,11 @@ $(function () {
   // Support full-text-search
   (function () {
     var query;
-    var relHref = $("meta[property='docfx\\:rel']").attr("content");
-    if (relHref) {
-      var search = searchFactory();
-      search();
-      highlightKeywords();
-      addSearchEvent();
-    }
+    var relHref = $("meta[property='docfx\\:rel']").attr("content") || "";
```

Comment:
`rel` always exists in our default template. It seems a bug that jQuery return null when `rel`'s value is `""`